src/main/java/gregtech/common/items/behaviors/TricorderBehavior.java

```diff
@@ -148,6 +148,22 @@ public List<ITextComponent> getScannerInfo(EntityPlayer player, World world, Blo
                                     metaTileEntity.getRegistry().getIdByObjectName(metaTileEntity.metaTileEntityId)))
                                             .setStyle(new Style().setColor(TextFormatting.BLUE))));
 
+            if (metaTileEntity.getOwner() != null) {
+                try {
+                    list.add(new TextComponentTranslation("behavior.tricorder.mte_owner",
```

Comment:
This could pull out the check for getting the player by UUID, to avoid the try catch